lib/config.js

```diff
@@ -1,3 +1,25 @@
-export const {
+const {
+  IE_CONTRACT_ADDRESS = '0x8460766Edc62B525fc1FA4D628FC79229dC73031',
+  // FIXME Add back chain.love either when it's online or once onContractEvent
+  // supports rpc failover
+  // RPC_URLS = 'https://api.node.glif.io/rpc/v0,https://api.chain.love/rpc/v1',
+  RPC_URLS = 'https://api.node.glif.io/rpc/v0',
+  GLIF_TOKEN,
   DATABASE_URL = 'postgres://localhost:5432/spark_stats'
 } = process.env
+
+const rpcUrls = RPC_URLS.split(',')
+const RPC_URL = rpcUrls[Math.floor(Math.random() * rpcUrls.length)]
+console.log(`Selected JSON-RPC endpoint ${RPC_URL}`)
+
+const rpcHeaders = {}
+if (RPC_URL.includes('glif')) {
+  rpcHeaders.Authorization = `Bearer ${GLIF_TOKEN}`
+}
+
+export {
+  IE_CONTRACT_ADDRESS,
+  RPC_URL,
+  DATABASE_URL,
+  rpcHeaders
+}
```

Comment:
It's a good idea to have `lib/config.js`, please move this new code to `observer/lib/config.js` and clean up `observer/bin/spark-observer.js` accordingly.